Tasks/TwineAuthenticateV1/twineauthenticatemain.ts

```diff
@@ -72,6 +139,26 @@ function findDuplicatesInArray<T>(array: Array<T>): Array<T>{
     return array.filter((e, i, a) => a.indexOf(e) !== i);
 }
 
+function removeFromFeedCount(internalFeed: auth.AuthInfo[], externalEndpoints: auth.AuthInfo[], entry: auth.AuthInfo): void {
+    if (internalFeed.includes(entry)) {
+        internalFeed.pop();
+        return;
+    }
+    externalEndpoints.pop();
+}
+
+function getNestedRepoProperty(connection: object): string {
+    for (const key in  connection) {
+        if (typeof connection[key] === 'object') {
+            return getNestedRepoProperty(connection[key]);
+        }
+        else if (key === 'repository') {
```

Comment:
this is a case sensative check please confirm the repository key is always lowercase?

Reply:
Changed it to key.toLowerCase( )!